flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/join/AbstractTwoInputStreamOperatorWithTTL.scala

```diff
@@ -51,11 +59,11 @@ abstract class AbstractTwoInputStreamOperatorWithTTL(
   private val maxRetentionTime: Long = queryConfig.getMaxIdleStateRetentionTime
   private val stateCleaningEnabled: Boolean = minRetentionTime > 1
 
-  private val ACTIVE_CLEANUP_TIMESTAMP = "cleanup-timestamp"
+  private val CLEANUP_TIMESTAMP = "cleanup-timestamp"
   private val TIMERS_STATE_NAME = "timers"
 
   // the latest registered cleanup timer
-  private var cleanupTimeState: ValueState[JLong] = _
+  private var cleanUpTimeState: ValueState[JLong] = _
```

Comment:
rename variable to `latestRegisteredCleanUpTimer` and drop the comment. Name `cleanUpTimeState` doesn't tell much.